docs/pages/observation_period/observation_period.md

```diff
@@ -0,0 +1,16 @@
+---
+title: OBSERVATION_PERIOD
+keywords: observation_period, observation_end_date
+last_updated: May 04, 2024
+tags: [observation_period, event, survival]
+summary: "Conventions related to resolving OBSERVATION_PERIOD table."
```

Comment:
would match other table summaries here "Conventions related to mapping data into the OBSERVATION_PERIOD table."